doc/guides/maintaining-openssl.md

```diff
@@ -2,6 +2,18 @@
 
 This document describes how to update `deps/openssl/`.
 
+If you need to provide updates across all active release lines you will
+currently need to generate two PRs as follows:
```

Comment:
This time round we did three PRs -- one for current (including the QUIC patches), one for 14.x (without the the QUIC patches) and one for 10.x.

Reply:
k will update for that.

Reply:
@richardlau updated